03week/ticTacToe.js

```diff
@@ -3,91 +3,155 @@
 const assert = require('assert');
 const readline = require('readline');
 const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 let board = [
-  [' ', ' ', ' '],
-  [' ', ' ', ' '],
-  [' ', ' ', ' ']
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
 ];
 
 let playerTurn = 'X';
+let playerOne = 'X';
+let playerTwo = 'O';
 
 function printBoard() {
-  console.log('   0  1  2');
-  console.log('0 ' + board[0].join(' | '));
-  console.log('  ---------');
-  console.log('1 ' + board[1].join(' | '));
-  console.log('  ---------');
-  console.log('2 ' + board[2].join(' | '));
+    console.log('   0  1  2');
+    console.log('0 ' + board[0].join(' | '));
+    console.log('  ---------');
+    console.log('1 ' + board[1].join(' | '));
+    console.log('  ---------');
+    console.log('2 ' + board[2].join(' | '));
 }
 
 function horizontalWin() {
```

Comment:
There's a lot of repetition in these win functions. Try to simplify the logic in these.